gitversioner/build.gradle

```diff
@@ -0,0 +1,43 @@
+apply plugin: 'kotlin'
+apply plugin: 'com.novoda.bintray-release'
+
+group = 'com.pascalwelsch.gitversioner'
+version = '0.0.1-SNAPSHOT'
+
+buildscript {
+    ext.kotlin_version = '1.1.2-4'
+    configure([repositories, project.repositories]) {
+        maven { url 'https://repo.gradle.org/gradle/repo' }
```

Comment:
As the `kotlin-gradle-plugin` is available via the gradle plugin portal you can have a much much easier implementation of the plugin:
```groovy
plugins {
  id "org.jetbrains.kotlin.jvm" version "1.1.2-2"
}
```
See https://plugins.gradle.org/plugin/org.jetbrains.kotlin.jvm